configs/orangepi/lite/scripts/install_sd.sh

```diff
@@ -54,20 +54,25 @@ boot_dir="${WORK_DIR}/boot"
 rootfs_dir="${WORK_DIR}/rootfs"
 persist_dir="${WORK_DIR}/persist"
 
+ORANGEPI_SD_SFX=$ORANGEPI_SD
+if [ -b ${ORANGEPI_SD}p1 ]; then
```

Comment:
There must be a better way to detect this

Reply:
```
ORANGEPI_SD_SFX1=lsblk -Po NAME $ORANGEPI_SD | sed -n -e '2 s/NAME="\(.*\)"/\1/p'
```
is way too complex IMO, and I don't have better idea :/

Reply:
It was a bit of a hack before.